rust/protocol/src/identity_key.rs

```diff
@@ -15,31 +24,39 @@ use prost::Message;
 const ALTERNATE_IDENTITY_SIGNATURE_PREFIX_1: &[u8] = &[0xFF; 32];
 const ALTERNATE_IDENTITY_SIGNATURE_PREFIX_2: &[u8] = b"Signal_PNI_Signature";
 
+/// The public identity of a user, used in [IdentityKeyStore].
+///
+/// Wrapper for [PublicKey].
 #[derive(Debug, PartialOrd, Ord, PartialEq, Eq, Clone, Copy)]
 pub struct IdentityKey {
     public_key: PublicKey,
 }
 
 impl IdentityKey {
+    /// Initialize a public-facing identity from a public key.
     pub fn new(public_key: PublicKey) -> Self {
         Self { public_key }
     }
 
+    /// Return a public key representing the public identity.
```

Comment:
Nitpick: I'd say "the public key" and "this identity". One could imagine a system where the same public identity would have multiple keys, but once you've already gotten to an IdentityKey, there's exactly one PublicKey that goes with it.

Reply:
Done! Thanks so much for the explanation!